.github/workflows/kind-e2e.yaml

```diff
@@ -18,10 +18,10 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         cluster:
-        - v1.20.7/contour
         - v1.21.1/kourier
         - v1.22.2/istio
-        - v1.22.2/gateway-api
+        - v1.23.0/istio
```

Comment:
Could you replace this and L#53 with `contour`? v1.22.2 is also istio and we miss contour now.